Dockerfile.ubuntu

```diff
@@ -0,0 +1,207 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG BASE_IMAGE=ubuntu:20.04
+FROM $BASE_IMAGE as base_build
+
+LABEL version="1.0.0"
+
+ARG PROJECT_NAME="OpenVINO Model Server 2021.4"
```

Comment:
2022.1?

Reply:
I think we will change it later on in all dockerfiles.